File: apps/commerce-api/src/main/java/com/loopers/infrastructure/commerceevent/CommerceEventJpaRepository.java
package com.loopers.infrastructure.commerceevent;

import java.util.List;

import org.springframework.data.jpa.repository.JpaRepository;

import com.loopers.domain.commerceevent.CommerceEvent;
import com.loopers.domain.commerceevent.CommerceEventStatus;

public interface CommerceEventJpaRepository extends JpaRepository<CommerceEvent, Long> {
    List<CommerceEvent> findAllByStatus(CommerceEventStatus commerceEventStatus);
}

File: apps/commerce-streamer/src/main/kotlin/com/loopers/infrastructure/commerceevent/CommerceEventJpaRepository.kt
package com.loopers.infrastructure.commerceevent

import com.loopers.domain.commerceevent.CommerceEvent
import org.springframework.data.jpa.repository.JpaRepository

interface CommerceEventJpaRepository : JpaRepository<CommerceEvent, Long> {
    fun findByEventId(eventId: String): CommerceEvent?
}
